tests/chainer_tests/training_tests/test_trainer.py

```diff
@@ -11,6 +11,19 @@
 from chainer import training
 
 
+class DummyExtension(training.extension.Extension):
+
+    def __init__(self):
+        self.is_called = False
+        self.is_finalized = False
+
+    def __call__(self, trainer):
+        self.is_called = True
+
+    def finalize(self):
+        self.is_finalized = True
+
+
 class TestTrainerElapsedTime(unittest.TestCase):
```

Comment:
Now that this class not only for testing elapsed time, could you rename the class name, say, to `TestTrainer`?